README.md

```diff
@@ -8,7 +8,7 @@
 
 - [📗 Table of Contents](#-table-of-contents)
 - [📖 OOP School Library ](#-School Library-)
-    - [How to build the Decode-Morse-Code"](#how-to-build-the-School Library)
+    - [How to build the OOP School Library"](#how-to-build-the-School Library)
   - [🛠 Walk through ](#-walk-through-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
```

Comment:
- [ ] [OPTIONAL] I'd suggest you fix the first link on the table of contents as it's being displayed as normal text instead of a section link as shown below 👇 which is making your repo look unprofessional 👶
  ![image](https://github.com/Bettenoch/OOP-School-Library/assets/104968236/ca638365-138e-4741-a444-db8c21d34ffc)